package.json

```diff
@@ -40,7 +40,7 @@
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-qunit": "^4.1.1",
-    "ember-cli-sass": "^7.1.7",
+    "ember-cli-sass": "^8.0.1",
```

Comment:
I didn't see a changelog on the ember-cli-sass repo.  What are the major differences between this version and our previous?

Reply:
The major change with 8.x was to switch away from `node-sass` to a pure JS implementation of `sass` (named accordingly). The ember-cli-sass blueprint would add this to the project's dependencies and use that by default unless we provide an alternative implementation.

Reply:
See https://github.com/aexmachina/ember-cli-sass/commit/a8fbf2e3e5aa6d3ef9907a4dfa1af9fc258353fa#diff-118ca7ee7bc1f499d2017a1323588431

Reply:
8.0 changed the implementation but forgot to add the `sass` dependency and this commit fixed that in 8.0.1